opencog/nlp/relex2logic/declarative.scm

```diff
@@ -0,0 +1,50 @@
+(define declarative
+    (BindLink
+        (VariableList
+            (TypedVariableLink
+                (VariableNode "$word-inst-node")
+                (TypeNode "WordInstanceNode")
+            )
+            (TypedVariableLink
+                (VariableNode "$parse-node")
+                (TypeNode "ParseNode")
+            )
+            (TypedVariableLink
+                (VariableNode "$interp-node")
+                (TypeNode "InterpretationNode")
+            )
+        )
+        (AndLink
+            (LemmaLink
+                (VariableNode "$word-inst-node")
+                (WordNode ".")
```

Comment:
Isn't there any other structure than a `.` to identify a declarative sentence? In a chat-bot context we can't be confident about punctuations, right? @ruiting @bgoertzel any suggestions.